site/en/docs/extensions/reference/userscripts/index.md

```diff
@@ -0,0 +1,102 @@
+---
+api: userScripts
+---
+{% Aside 'note' %}
+This API is currently available in Chrome 120 beta.
+{% endAside %}
+## Manifest
+
+To use the `chrome.userScripts` API, add the `"userScripts"` permission to your manifest.json and `"host_permissions"` for sites you want to run scripts on.
+
+```json/9-15
```

Comment:
Can we fix the highlighted lines or remove them?

Reply:
Fixed.